ansible/README.md

```diff
@@ -58,9 +58,10 @@ As another alternative you can probably install ansible into your virtualenv (un
 Examples
 --------
 
-In the following examples replace example-hosts with the private host inventory file
+In the following examples replace example-hosts with the private host
+inventory file
```

Comment:
Cheers. Didn't want to get into a situation where everyone's doing different things. Rest of the PR looks OK